### ISSUE TITLE
Feature request: writing PR comments in batch

### ISSUE THREAD

**Issue by Martinsos** [isaacs/github#1276]
When reviewing PRs, typical workflow I go through is: Reviewer writes comments, when he wrote all of them, author or PR checkes them, answers all of them, commits fixes, then notifies the reviewer to take another look, and that repeats in a loop until everybody is satisfied.

Big thing for me when reviewing code is ability to use "Review changes" feature, which allows me to write comments in a batch, meaning they are all submitted only when I submit the whole review and are stored until I do so, as a draft. Before this feature was here, I had to write comment by comment, it is unclear to author of PR if I am done or not, and often I would have to go back and delete or edit comments because I learned later in the code something new.

However, author of PR still has to write comment by comment, instead of writing them in a batch fashion, same like reviewer can do now with "Review changes" feature. Main problem here is that reviewer never knows when author is done with comments, since they are coming one by one, so author needs to somehow let reviewer know that he is done with everything and that he should take another look.
Also, if author figure smt out from later comments that reflects his previous answers, he has to go and edit / delete those, and maybe author received email with those and so on.

**TLDR:**
So what I would love to have, is this ability for author of PR to send comments in a batch and have them saved as a draft in the meantime, exactly in the same fashion as "Review changes" feature is working right now for the reviewer.

 did a search through the existing issues and could not find anything similar, so I hope I did not miss anything.
Let me know if you also think this is useful, or if you have suggestions how you handle this right now.

**Comment by MNF** [comment]
@Martinsos The problem you described is correct if an author is using main “conversation” tab to reply to each reviewer’s comment.
As a workaround author can switch to  “Files changed” tab, and there is an option to keep all comments as “pending” and then submit all of them at once(simulating review of own PR)

It’s definitely not intuitive, and I would appreciate if GitHub will implement it properly for “conversation” tab

**Comment by Martinsos** [comment]
@MNF thanks for the explanation! I actually believe this feature did not exist at the point when I opened this issue, because recently I have been using the feature you described - so either I just didn't know about it at the point of opening this issue, or it didn't exist. In any case, I will close it as solution exists!

**Comment by MNF** [comment]
@Martinsos , I don't think, that you should close the suggestion. 
The use review section for post-review replies is a not-intuitive workaround, and GitHub should support pending comments on main conversation tab as well.
